contentcuration/contentcuration/static/js/edit_channel/clipboard/hbtemplates/clipboard_add_folder.handlebars

```diff
@@ -0,0 +1,19 @@
+<div id="clipboard">
+	<div class="clipboard_header">
+		<h3 class="text-center">
+			<img src="/static/img/add_folder.png" alt="add" class="pull-left"/>
```

Comment:
The 'add_folder' icon looks like a file icon with a plus, rather than a folder.

Also, is there a reason to use a png file rather than the Bootstrap glyphicons?